doc/nrf/ug_nrf5340.rst

```diff
@@ -56,10 +56,11 @@ Use this core for tasks that require high performance and for application-level
 The M33 TrustZone divides the application MCU into secure and non-secure domains.
 When the MCU boots, it always starts executing from the secure area.
 
-In Zephyr, the application core is divided into two different build targets:
+In Zephyr, the application core is built using one of the following build targets:
 
-* ``nrf5340dk_nrf5340_cpuapp`` for the secure domain
-* ``nrf5340dk_nrf5340_cpuapp_ns`` for the non-secure domain
+* ``nrf5340dk_nrf5340_cpuapp`` for the secure domain.
+* ``nrf5340dk_nrf5340_cpuapp_ns`` for the non-secure domain.
```

Comment:
for firmware running in the non-secure domain

Reply:
As we have already mentioned in the main sentence (line 59) about the firmware, I feel it is not required here.